.github/workflows/danger.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - release/**
```

Comment:
Don't you need to update all the other workflows as well?

Reply:
No, Craft will create a `release/{version}` branch and take the artifacts from the CI of that branch. See the first step in https://github.com/getsentry/craft#integrating-your-project-with-craft.

Reply:
The first step mention that:

> Set up a workflow that builds your assets and runs your tests. Allow building release branches (their names follow release/{VERSION} by default, configurable through releaseBranchPrefix).

Here you're editing the `danger.yml` workflow that is just checking that the release notes are correct. I believe you should include the `release/**` branch rule on also the other workflows that are actually running tests (e.g. `pre-merge.yml` and others). Or am I missing something? :)

Reply:
Yes, you're right. Good catch, updated.